todo_app/templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{item.id}} - {{item.status}} - {{item.title}}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
-{% endblock %}
\ No newline at end of file
+
+  <form action="/add"  method="post">
+   <label for="Title">Task Title:</label><br>
+    <input type="text" id="title" name="title"><br>
```

Comment:
Really minor, but adding a `required` keyword to this input will help users not accidentally submit an empty to-do
```suggestion
<input type="text" id="title" name="title" required><br>
```